src/tabs/common/css_constants.ts

```diff
@@ -0,0 +1,25 @@
+/* [Imports] */
```

Comment:
For these constants, would it be possible to grab them from some `@blueprintjs` export? For example `export { SA_TAB_BUTTON_WIDTH } from '@blueprintjs/core';`?

I'm working on externalizing all the packages that are available at runtime, so if this is possible we might not need to hardcode these.

Reply:
I wasn't able to find such exported constants in BlueprintJS. The most I could find was IconSize, because that's provided by the library to use for the... icon's size property. That's why I resorted to extracting the values I wanted from the actual webpage myself, and named them accordingly.

Also, not all of them come from Blueprint. Some of them appear to be custom frontend CSS, and there's also stuff from the Ace editor theme that I wanted to use, for example for the canvas background colour to give a similar look. So either way, I think this file would still exist. But of course, if Blueprint does provide us with some of these values, I would prefer to import it from them instead.

Reply:
Perhaps we should have some kind of mechanism for such constants from the frontend into modules?

Reply:
That does sound good.

Reply:
I will move this discussion to a frontend issue shortly. For now, I think this CSS constants file will do.